package.json

```diff
@@ -23,7 +23,9 @@
     "clean": "rm -rf src/*/node_modules/ build/ node_modules/ src/server_manager/install_scripts/do_install_script.ts src/server_manager/install_scripts/gcp_install_script.ts third_party/shellcheck/download/",
     "lint": "npm run lint:sh && npm run lint:ts",
     "lint:sh": "bash ./scripts/shellcheck.sh",
-    "lint:ts": "npx tslint 'src/**/*.ts' -e '**/node_modules/**'"
+    "lint:ts": "npx tslint 'src/**/*.ts' -e '**/node_modules/**'",
+    "start": "npm run action start",
```

Comment:
I acknowledge that the command becomes a little shorter, abut it's not like we need to type it every time.

I have a few concerns:
- This doesn't actually work like what developers will expect. `npm start` should just start the main for the package, with the rest being the commandline arguments.
- With workspaces, developers would expect to pass the workspace of the package you are starting
- This makes the run action more complex by introducing the concept of a scope and creates inconsistencies based on the action you are running.

We may want to punt on this until we figure out how to deal with multiple modules and think it through.
Things would change. For instance, if each module has its own package.json, then we can express dependencies and do things like `npm -w server/integration_test start`